packages/backend/src/graphql/resolvers/incoming_payment.ts

```diff
@@ -94,7 +97,8 @@ export const createIncomingPayment: MutationResolvers<ApolloContext>['createInco
         ? undefined
         : new Date(args.input.expiresAt),
       incomingAmount: args.input.incomingAmount,
-      metadata: args.input.metadata
+      metadata: args.input.metadata,
+      tenantId: ctx.tenant.id
```

Comment:
```suggestion
      tenantId: ctx.forTenantId
```
This allows us the operator to create incoming payments for a specific tenant they've specified in the `tenantId` mutation input